crates/tx-parser/src/filter.rs

```diff
@@ -24,6 +25,77 @@ pub enum TxFilterRule {
     DepositRequest(DepositTxParams),
     /// Deposit transaction with deposit config and address
     Deposit(DepositTxParams),
+    /// Addresses that are spent to
+    SpentToAddrs(Vec<BitcoinAddress>),
+    /// Blob ids that are expected
+    BlobIds(Vec<Buf32>),
+    /// Outpoints
+    Outpoints(Vec<(Buf32, u32)>),
+}
+
+#[derive(Clone, Debug)]
+pub struct TxFilterConfig {
+    /// For checkpoint update inscriptions.
+    rollup_name: RollupName,
+
+    /// For addresses that we expect spends to.
+    expected_addrs: Vec<BitcoinAddress>,
+
+    /// For blobs we expect to be written.
+    expected_blobs: Vec<Buf32>,
+
+    /// For deposits that might be spent from.
+    expected_outpoints: Vec<(Buf32, u32)>,
+    // eventually, in future version
+    // For bridge V1 deposits that do bitmap flagging for the multisig addr.
+    // operator_key_tbl: PublickeyTable,
+    /// EE addr length
+    ee_addr_len: u8,
+
+    /// Deposit denomination
+    deposit_denomination: u64, // sats
```

Comment:
Don't bake this in as a single value, it should be capable of being extended to support multiple denominations easily even if we don't support this today.

Reply:
Let's do that when we have concrete decisions/specs on what kind of denominations we will have.